app/assets/stylesheets/application.scss

```
@@ -0,0 +1,172 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ */
+
+/* Custom bootstrap variables must be set or imported *before* bootstrap. */
+@import "bootstrap";
+
+@import "**/*";
+
+ /* ====================================================================================================================
+ * WEB-LAYOUT STYLES
+ * ====================================================================================================================*/
+
+body {
+    background-color: white;
+}
+
+header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: right;
+    background-color: black;
+    font-family: 'Poppins', sans-serif;
+    color: white;
+    border-bottom: 1em solid grey;
+    background-image: url(image-path('car-icon-blue.png'));
+    background-position: left;
+    background-repeat: no-repeat;
+    background-size: 200px;
+    height: 15vh;
+    min-height: 150px;
+}
+
+header div.header-spacer {
+    width: 200px;
+}
+
+header h1 {
+    padding: 1em;
+}
+
+nav {
+    text-align: right;
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.25em;
+}
+
+section.flash div.green-message {
```

Your class names shouldn't talk about what style they produce.

For example, if you decided to change your success indicator to be an icon instead of a color this would no longer make sense as a class name.

Something like `success-message` would be preferable:

```suggestion
section.flash div.success-message {
```